src/components/Calculator.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  render() {
+    return (
+      <>
+        <table cellSpacing="0">
+          <tr>
+            <td colSpan="3"><input type="text" value="0" id="result" /></td>
+          </tr>
+          <tr>
+            <td>
+              <input className="others" type="button" value="AC" />
+              {' '}
+            </td>
+            <td>
+              <input className="others" type="button" value="+/-" />
+              {' '}
+            </td>
+            <td>
+              <input className="others" type="button" value="%" />
+              {' '}
+            </td>
+            <td>
+              <input className="operants" type="button" value="/" />
+              {' '}
+            </td>
+          </tr>
```

Comment:
- [OPTIONAL] 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻